docs/source/setup/single-machine.rst

```diff
@@ -35,11 +35,11 @@ use this scheduler, however you do have a choice between threads and processes:
 Selecting Threads, Processes, or Single Threaded
 ------------------------------------------------
 
-Currently these options are available by selecting different ``get`` functions:
+Currently, these options are available by selecting different ``get`` functions:
 
--  ``dask.threaded.get``: The threaded scheduler
--  ``dask.multiprocessing.get``: The multiprocessing scheduler
--  ``dask.local.get_sync``: The single-threaded scheduler
+-  ``dask.threaded.get``: The threaded scheduler.
+-  ``dask.multiprocessing.get``: The multiprocessing scheduler.
+-  ``dask.local.get_sync``: The single-threaded scheduler.
```

Comment:
I tend not to end phrases like these with periods because they are not full sentences.

Reply:
Before deciding which one to use at first, I've dug in the documentation a bit to see which way was more used, and I've found several conflicting occurrences of how numbered lists are terminated (or not) in the documentation. For example, here are some samples of lists terminated with dots:

https://dask.pydata.org/en/latest/index.html#dask

https://dask.pydata.org/en/latest/support.html#discussion

https://dask.pydata.org/en/latest/support.html#asking-for-help

https://dask.pydata.org/en/latest/scheduling.html#scheduling

and the opposite as well:

https://dask.pydata.org/en/latest/spec.html#definitions

https://dask.pydata.org/en/latest/configuration.html#configuration

or even mixed use cases:

https://dask.pydata.org/en/latest/dataframe.html#common-uses-and-anti-uses

It was conflicting for me to decide which one to use so I've picked the one which made more sence to me at first. 

However, after pondering a bit about this, and taking a look at other documentations of other projects, I came to the conclusion that it is best to avoid terminating sentences altogether and these changes (and others before this one) should be reverted / corrected to remove punctuation at the end of the sentence.